honeybee_radiance_command/gensky.py

```diff
@@ -0,0 +1,228 @@
+"""gensky command."""
+
+from .options.gensky import GenskyOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+
+
+class Gensky(Command):
+    """gensky command."""
+
+    __slots__ = ('_month', '_day', '_time', '_time_zone', '_solar_time', '_input')
+
+    def __init__(self, month, day, time, time_zone=None, solar_time=False,
```

Comment:
How would you handle a case that user inputs `-ang altitude azimuth`? Should we allow month, day and time to be None for such a case?